path-finding.asciidoc

```diff
@@ -93,13 +93,15 @@ She would find out that Bob would not forward the onion, if she believed that sh
          "channel": "357",
          "direction": 1,
          "satoshi": 103000,
+         "forward": 102000,
```

Comment:
not sure if that is correct. 
at the time I was writing this the API did not specify the forward amount in the onion. and also at other points we just have the amt_to_forward how I interpreted the `satoshi` field.